src/Microsoft.Build.UnGAC/Program.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Build.UnGAC
+{
+    class Program
```

Comment:
Does this work, and is this the right place for this? I'm surprised first that this runs at install time rather than running every time the user builds and second that you had to make a new project to make something that runs automatically. Could you explain that a little more?

Reply:
What will allow it to run at install time is the `exe` being pointed to from `src/Package/MSBuild.VSSetup/exe.swr`, which `src/Package/MSBuild.VSSetup/files.swr` will point at. And `files.swr` is already being picked up by microbuild.